repo/packages/H/hdfs/0/command.json

```diff
@@ -0,0 +1,5 @@
+{
+  "pip": [
+    "git+https://github.com/mesosphere/dcos-hdfs.git#dcos-hdfs=0.1.0"
```

Comment:
This is okay for now but be aware that we should eventually install a well known versions. I would recommend adding this project to TeamCity to have it generate wheels. I think that @BenWhitehead is doing something similar so talk to him if you have questions.